layouts/_default/list.html

```diff
@@ -16,6 +16,15 @@
         <h2 class="post__title">
           <a href="{{.RelPermalink}}">{{ .Title }}</a>
         </h2>
+        {{if .Site.Params.blog.summary }}
```

Comment:
Since this is the `_default` layout, we should use site-wide configs

Reply:
Can replace blog as list then? but readability will be reduced?

Reply:
I mean we can just use `.Site.Params.showSummary`

Reply:
```suggestion
        {{if .Site.Params.blog.summary }}
```
Updated.